src/tally.cpp

```diff
@@ -536,32 +558,32 @@ std::string pyne::Tally::form_mcnp_meshtally(
   } else if (entity_geometry.find("Cylinder") != std::string::npos) {
     mtally_stream << "CYL" << std::endl;
     if (!is_zero(axs)) {
-      mtally_stream << indent_block << "AXS=" << pyne::join_to_string(axs) << std::endl;
+      mtally_stream << indent_block << "AXS= " << pyne::join_to_string(axs) << std::endl;
```

Comment:
Why this space?  I think these are better without the space.

Reply:
I changed the join_to_string method, to not have the delimiter before the first joined item...
the test included the space, so I added it back here.

I will remove that one and fix the test accordingly